tests/test_modifiers.py

```diff
@@ -35,7 +35,7 @@ def test_removing_indentation(self, create_instance: Modifiers):
         assert value.file_contents[1].startswith(FOUR_SPACES)
         assert value.modified_contents == [
             "def say_hello() -> str:\n",
-            '  return "Hello!"\n',
+            '  if 1 == True and 0 == False: return "Hello!"\n',
```

Comment:
`'  return "Hello!"\n',`

Reply:
Just after the last `@pytest.fixture`, you can include:
```py
@pytest.fixture
def create_instance_with_boolean():
    boolean_input = ["    if 1 == True and 0 == False:\n", "        return 'Hello!'\n"]
    yield Modifiers(test_input[:1] + boolean_input + test_input[2:])
```

Reply:
Thanks for help with these changes